pkg/graph/schema.graphql

```diff
@@ -1225,6 +1229,39 @@ enum SystemIntakeGRBReviewerVotingRole {
   NON_VOTING
 }
 
+type SystemIntakeGRBReviewDiscussionPost {
+  id: UUID!
+  content: HTML!
+  votingRole: SystemIntakeGRBReviewerVotingRole
+  grbRole: SystemIntakeGRBReviewerRole
+  systemIntakeID: UUID!
+  createdByUserAccount: UserAccount!
+  createdAt: Time!
+  modifiedByUserAccount: UserAccount
+  modifiedAt: Time
+}
+
+type SystemIntakeGRBReviewDiscussion {
+  initialPost: SystemIntakeGRBReviewDiscussionPost!
+  replies: [SystemIntakeGRBReviewDiscussionPost!]!
+}
```

Comment:
The idea here was to not have two different post types but still organize them so that they're grouped into threads.